### PR TITLE
Flickering protractor test

### PR DESCRIPTION
**Pull request by hschink**
This implements @0xF013's and @myabc's idea (see https://github.com/opf/openproject/pull/2286#issuecomment-65383341 and https://github.com/opf/openproject/pull/2286#issuecomment-65384758 for details).

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage. <a href='http://fix-flickering-protractor-test.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Click to open.</a>

**Comment by myabc** [comment]
For reference, alt. solution: https://gist.github.com/myabc/f5e153cdf2f7d45e04d0

File: frontend/tests/integration/work-package-details-spec.js [hunk 1]
```diff
@@ -37,10 +37,12 @@ var WorkPackageDetailsPane = require('./pages/work-package-details-pane.js');
 /*jshint expr: true*/
 
 describe('OpenProject', function() {
-  var page = new WorkPackageDetailsPane(819, 'overview');
+  before(function() {
+    var page = new WorkPackageDetailsPane(819, 'overview');
```
Comment (myabc) [inline frontend/tests/integration/work-package-details-spec.js]: @hschink can you wrap this first test in a `context`?

Comment (hschink) [inline frontend/tests/integration/work-package-details-spec.js]: @myabc I amended the change to 9c63c36.

**Comment by myabc** [comment]
@hschink I rebased the Angular 1.3 branch (#2279) against dev after merging this PR. Unfortunately, it appears the issue reappears there: https://travis-ci.org/opf/openproject/jobs/42979896

**Comment by hschink** [comment]
@myabc Ok, I guess you'll then check your approach https://github.com/opf/openproject/pull/2293#issuecomment-65450122, right?

**Pull request merged**